packages/web/src/components/FormButton.js

```diff
@@ -26,22 +25,20 @@ function FormButton(props) {
 
 	return (
 		<Button
-			isDisabled={isDisabled || !isValid || !isTouched}
-			type={type}>
+			{...props}
+			isDisabled={isDisabled || !isValid || !isTouched}>
 			{children}
 		</Button>
 	)
 }
 
 FormButton.defaultProps = {
 	isDisabled: false,
-	type: 'button',
```

Comment:
I removed the `type` defaults and assignment because it's handled in the `<Button>` component.